packages/db/lib/migrator.mjs

```diff
@@ -45,14 +56,31 @@ async function execute (logger, args, config) {
       'migration-finished',
       (migration) => logger.debug(`completed ${basename(migration.filename)}`)
     )
+    const migrations = await postgrator.getMigrations()
+    const current = await postgrator.getDatabaseVersion()
+
+    if (args.rollback) {
+      if (current === 0) {
+        logger.info('No migrations to rollback')
+        return
+      }
+      let nextDown = migrations.filter(x => x.version < current)
+        .filter(x => x.action === 'undo')
+        .sort((a, b) => a.version - b.version).reverse()
+        .map(extractVersionFromMigration)[0]
```

Comment:
Why do we need to apply the map to all migrations if we need only one?

Reply:
It's just cleaner in case don't have migrations to rollback to. Otherwise we should check if we don't have a migration before (or inside) `extractVersionFromMigration`. Also `extractVersionFromMigration` is just a temporary workaround for [this postgrator bug](https://github.com/rickbergfalk/postgrator/issues/160), when fixed we can get rid of this (ugly) function and of the `map`